service-rpc/src/main/thrift/rpc.thrift

```diff
@@ -299,7 +299,9 @@ service TSIService {
 
 	TSStatus createTimeseries(1:TSCreateTimeseriesReq req);
 
-  TSStatus deleteTimeseries(1:list<string> path)
+    TSStatus deleteTimeseries(1:list<string> path)
+
+    TSStatus deleteStorageGroup(1:TSDeleteDataReq req);
```

Comment:
```suggestion
    TSStatus deleteStorageGroup(1:string storageGroup);
```

Reply:
I think it is best if you make it a list.